.github/workflows/github-container-registry.yml

```diff
@@ -0,0 +1,169 @@
+##########################################################################
+#
+# WARNING: This file was generated by update.php.
+#
+# You can find the relevant template in the `/templates` folder.
+#
+name: Build test images
+
+on:
+  pull_request:
+
+env:
+  REGISTRY_USERNAME: desrosj
+  REGISTRY_PASSWORD: ${{ secrets.GHCR_TOKEN }}
+  PACKAGE_REGISTRY_HOST: ghcr.io
+  PACKAGE_REGISTRY: ghcr.io/wordpress/wpdev-docker-images
+  PR_TAG: -${{ github.event.number }}
+
+
+jobs:
+  # Checks for any incomplete preceeding workflow runs and cancels them.
+  check-for-duplicates:
+    name: Cancel duplicate workflow runs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+  build-php-images:
+    name: PHP ${{ matrix.php }} Image
+    runs-on: ubuntu-latest
+    needs: check-for-duplicates
+    strategy:
+      matrix:
+        php: [ '5.2', '5.3', '5.4', '5.5', '5.6.20', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+    env:
+      PHP_VERSION: ${{ matrix.php }}
+      PHP_LATEST: '7.4'
+
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
```

Comment:
No need to do this because of `needs: check-for-duplicates`.

Reply:
Fixed!